src/auro-datepicker.js

```diff
@@ -716,20 +729,21 @@ export class AuroDatePicker extends LitElement {
               </auro-input>
             ` : undefined}
           </div>
-          <div class="calendarWrapper">
+          <div class="calendarWrapper" part="calendarWrapper">
             <auro-calendar
               ?noRange="${!this.range}"
               .min="${this.convertToWcValidTime(new Date(this.minDate))}"
               .max="${this.convertToWcValidTime(new Date(this.maxDate))}"
               .maxDate="${this.maxDate}"
               .minDate="${this.minDate}"
+              part="calendar"
             >
               <slot slot="mobileDateLabel" name="mobileDateLabel"></slot>
               <span slot="mobileDateFromStr">${this.value ? this.getMobileDateStr(this.value) : html`<span class="placeholderDate">MM/DD/YYYY</span>`}</span>
               ${this.range ? html`<span slot="mobileDateToStr">${this.valueEnd ? this.getMobileDateStr(this.valueEnd) : html`<span class="placeholderDate">MM/DD/YYYY</span>`}</span>` : undefined}
             </auro-calendar>
           </div>
-          <span slot="helpText">
+          <span slot="helpText" part="datepickerHelpText">
```

Comment:
Does this part name need to be this long? Can it just be `helpText`?

Reply:
This cannot just be `helpText` because there is another element within this `span` that has a csspart called `helpText`.